stripe/resource_stripe_product.go

```diff
@@ -28,6 +32,24 @@ func resourceStripeProduct() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"active": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  true,
+			},
+			// TODO Restore once we figure out how to properly delete this data with the Go client
+			//"attributes": &schema.Schema{
```

Comment:
I have opened https://github.com/stripe/stripe-go/issues/788 to get help with this. Either the client library is broken, or I am using it incorrectly.